manim/mobject/three_dimensions.py

```diff
@@ -134,6 +134,30 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
 
 
 class Sphere(ParametricSurface):
+    """A sphere
+
+    Examples
+    ---------
+    .. manim:: ExampleSphere
+        :save_last_frame:
+
+        class ExampleSphere(ThreeDScene):
+            def construct(self):
+                self.set_camera_orientation(phi = PI/6, theta = PI/6)
+                sphere1 = Sphere(center = (3,0,0),radius = 1,resolution = (30, 40),u_min = 0.001,u_max = PI - 0.001,v_min = 0,v_max = TAU)
+                sphere1.set_color(RED)
+                self.add(sphere1)
+                sphere2 = Sphere(center=(0,1,0),radius=2,resolution=(40, 50))
+                sphere2.move_to((-1,-3,0))
+                sphere2.set_color(GREEN)
+                self.add(sphere2)
+                sphere3 = Sphere(center=(0,1,0),radius=2,resolution=(40, 50))
```

Comment:
All these examples are pretty high res. I would recommend showing what a low res sphere would look like too. This is a sample value I toyed around with. 
```suggestion
                sphere3 = Sphere(center=(0,1,0),radius=2,resolution=(14, 14))
```

Reply:
This is a bit of an afterthought but: dropping the resolution of sphere1 to `(20, 30)` and sphere2 to `(35, 40)` would hardly affect their look and save a couple seconds in rendering time.

Reply:
@naveen521kk removed sphere from top line. Also copied the documentation from the bot. 

And also there is no hackathon label added to this thread.